.github/scripts/internationalization.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -x
+
+evalrc() {
+  if [ "$1" -ne "0" ]; then
+    echo "$2"
+    exit $1
+  fi
+}
+
+# This function converts candlepin version to a int value with (Zero) padding
+# which helps in comparing versions.
+# Examples -
+# 2.5.6  -> 2005006000
+# 3.2.0  -> 3002000000
+# 2.9.6  -> 2009006000
+get_simplified_version() {
+  echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }'
+}
+
+git fetch --all
+
+# Declare an associative array
+declare -A candlepin_java
+
+#Clean the project
+./gradlew clean
+evalrc $? "Project cleaning was not successful."
+
+#To generate the auto generated classes and files
+./gradlew war
+evalrc $? "Compilation was not successful for branch $GIT_BRANCH."
+
+#Execute the gettext task
+./gradlew gettext
+evalrc $? "Gettext was not successful for branch $GIT_BRANCH."
+
+files=$(git diff | egrep -v -e '^( |\+#|\-#|@@|\+\+\+|\-\-\-|diff|index)' -e 'X-Generator' -e 'POT-Creation-Date')
+
+if [ ! -z "$files" ]; then
+
+#Code to commit the updated template file
+git add po/keys.pot
+evalrc $? "Git add file was not successful for branch $GIT_BRANCH."
+
+echo "Committing and pushing the template file."
+git -c "user.name=$GIT_AUTHOR_NAME" -c "user.email=$GIT_AUTHOR_EMAIL" commit -m "updated po/keys.pot template"
+evalrc $? "Git commit was not successful for branch $GIT_BRANCH."
+
+git push https://${GITHUB_TOKEN}@github.com/candlepin/candlepin $GIT_BRANCH
+evalrc $? "Git push was not successful for branch $GIT_BRANCH."
+
+else
+#To stash the unnecessary changes
+git stash
```

Comment:
not sure what this is for:
- if nothing changed, there is nothing to stash anyway
- even if what changed is only the pot creation date, it does not matter anyway, as after this script runs the job ends

Reply:
Yeah, I'm not sure what the purpose of this is either. It was in the original script and may have been used for stashing changes before changing to another branch since the original script would add a commit for all the code line branches.